### ISSUE TITLE
Running KHEOPS on an M1 Mac

### ISSUE THREAD

**Issue by ktxk459** [OsiriX-Foundation/kheops#61]
I followed the [installation instructions](https://docs.kheops.online/docs/installation) for a local insecure server on my M1 Mac but for some reason I am not able to run it successfully. I was getting some errors related to the docker platform. Thus I set it using:
`export DOCKER_DEFAULT_PLATFORM=linux/amd64`

 However the UI shows this view and it is stuck here:

<img width="1365" alt="image" src="https://user-images.githubusercontent.com/95640166/235117321-863b3e44-5b1f-4223-816c-7234fcef1e31.png">

Trying to access keycloak at http://127.0.0.1:8080 I get nothing. 
Any suggestions?

**Comment by yuvraj-wale** [comment]
yes I was getting the same issue while trying to set up kheops on my m1 MacBook Air, currently it only seems to be available for linux or maybe windows.

**Comment by DennisMuchiri** [comment]
<img width="1368" alt="Screenshot 2023-07-13 at 11 06 57" src="https://github.com/OsiriX-Foundation/kheops/assets/11217100/969f12e9-f6a5-475e-a656-54c6ba3a53fb">
Please refer https://docs.kheops.online/docs/installation

**Comment by ksidiro** [comment]
Already tried those suggestions.

**Comment by DennisMuchiri** [comment]
@ksidiro okay.
